src/magicbox/MagicBox.ts

```diff
@@ -212,7 +209,6 @@ export class MagicBoxInstance {
   }
 
   private updateSuggestion(suggestions: Suggestion[]) {
-    this.lastSuggestions = suggestions;
     const firstSuggestion = this.getFirstSuggestionText();
     this.inputManager.setWordCompletion(firstSuggestion && firstSuggestion.text);
     this.onsuggestions && this.onsuggestions(suggestions);
```

Comment:
Why might `onsuggestions` not be a function? Also, should the name be camel-cased `onSuggestions`?

Reply:
> Why might `onsuggestions` not be a function? 

`onsuggestions` is set at the same time that `MagicBox` gets instantiated, so you're right that it's always a function. I'll remove this check.

> Also, should the name be camel-cased `onSuggestions`?

You're right, I don't see why it couldn't be camel-cased. I'll change that.